src/overlays/actors/ovl_player_actor/z_player.c

```diff
@@ -6434,16 +6434,16 @@ void func_8083D6EC(PlayState* play, Player* this) {
 }
 
 s32 func_8083DB98(Player* this, s32 arg1) {
-    Actor* unk_664 = this->unk_664;
+    Actor* lockOnActor = this->lockOnActor;
     Vec3f sp30;
     s16 sp2E;
     s16 sp2C;
 
     sp30.x = this->actor.world.pos.x;
     sp30.y = this->bodyPartsPos[PLAYER_BODYPART_HEAD].y + 3.0f;
     sp30.z = this->actor.world.pos.z;
-    sp2E = Math_Vec3f_Pitch(&sp30, &unk_664->focus.pos);
-    sp2C = Math_Vec3f_Yaw(&sp30, &unk_664->focus.pos);
+    sp2E = Math_Vec3f_Pitch(&sp30, &lockOnActor->focus.pos);
+    sp2C = Math_Vec3f_Yaw(&sp30, &lockOnActor->focus.pos);
     Math_SmoothStepToS(&this->actor.focus.rot.y, sp2C, 4, 10000, 0);
     Math_SmoothStepToS(&this->actor.focus.rot.x, sp2E, 4, 10000, 0);
     this->unk_6AE |= 2;
```

Comment:
This code specifically has had me skeptical on calling this "lockOnActor". It seems to be more generally just the actor that link should be looking at.

For example, `func_80851314` sets what will eventually be called the `csActor` to `unk_664`, to make link look at it in the cutscene. No lock on reticle involved.
Will do some more research (possibly document this function and the one it calls) for more info and get back to you. But just putting it out there

Reply:
It's also tied with talking as well, so I've reverted the name until more specifics are understood